cookbooks/symfony_no_flex.rst

```diff
@@ -0,0 +1,300 @@
+SymfonyCloud without Flex
+=========================
+
+SymfonyCloud experience is optimal when Symfony applications are using Flex, but
+not every project has been migrated to Flex yet. Even if you are using Symfony
+2.x or 3.x, you can start using SymfonyCloud and actually leverage some of its
+features to ease the migration to Symfony 4 and Flex.
+
+.. tip::
+
+   Although this guide helps you setuping your own project for SymfonyCloud, we
+   suggest you first follow the :doc:`Getting Started </getting-started>`
+   guide with a brand new Symfony 4 project and a free trial to get a better
+   understanding of SymfonyCloud features and philosophy.
+
+Initial setup
+-------------
+
+.. note::
+
+   You have to :ref:`install the CLI Tool <cli-install>` first.
+
+From within your project's directory, generate a sensible default SymfonyCloud
+configuration:
+
+.. code-block:: terminal
+
+    $ symfony project:init
+
+Thanks to our versatile templates most of the hard work of adapting the
+configuration to your Symfony project structure is already done
+for you in the ``.symfony.cloud.yaml`` and
+``app/config/parameters_symfonycloud.php`` files.
+
+As for any SymfonyCloud project, you will have to edit the
+``.symfony.cloud.yaml`` and ``.symfony/services.yaml`` files according to your
+needs. See the :doc:`Services documentation </services/intro>` for a complete
+description of currently supported service types, endpoints and associated
+environment variables.
+
+You will then have to tweak a bit the generated configuration. The instructions
+will mostly depend on your Symfony version.
+
+Symfony version
+---------------
+
+The quality of your experience with SymfonyCloud will vastly depend on your
+Symfony version: before Symfony 3.2 you do not benefit from the
+`Runtime Environment Variables`_ while the `DotEnv component`_ has been
+introduced with Symfony 3.3. SymfonyCloud leverages environment variables to
+expose services credentials and to allow fast creation of new environments with
+the same codebase. Those two features are key for a great SymfonyCloud
+experience.
+
+We highly recommend the upgrade to 3.2+ before getting started. If you cannot do
+so, your SymfonyCloud configuration will remain the same but your deployments
+will take longer as your Symfony cache will need to be built each time.
+
+.. tip::
+
+   As the configuration change between the two cases is minimal, you can
```

Comment:
changes

Reply:
actually, this is "the configuration change" that "is minimal".
or we can move to "As the configuration change**s** between the two cases **are** minimal" but that sounds weird